sensu/files/plugins/check-elastic.rb

```diff
@@ -71,10 +107,43 @@ def submit_alert(alert_string)
   end
 
   def run
+    if config[:critical] != nil || config[:warning] != nil
+      run_threshold_check
+    else
+      run_result_check
+    end
+  end
+
+  # This will raise an alert (that closes itself) when the thresholds are back
+  # below the limit.
+  def run_threshold_check
+    data = get_count
+    count = data['count']
+
+    if config[:out_string]
+      out = config[:out_string]
+    else
+      out = "#{config[:check]} #{count} records matched"
+    end
+
+    if config[:critical] != nil && count >= config[:critical]
+      critical "#{out} is above the CRITICAL limit: #{count} count / #{config[:critical]} limit"
+    elsif config[:warning] != nil && count >= config[:warning]
+      warning "#{out} is above the WARNING limit: #{count} count / #{config[:warning]} limit"
+    else
+      ok out
```

Comment:
If you mean `ok "#{out}"` then yes, as  `ok out` is equivalent to that but doesn't have a needless string interpolation.